### PR TITLE
ADEN-6537 fix-vuap-tests

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L155](https://github.com/Wikia/selenium-tests/blob/50ca7e68034e179b2a8978ef62fd2a62464f7ceb/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L155): Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)
1. ![MINOR][MINOR] [AutoplayVuap.java#L181](https://github.com/Wikia/selenium-tests/blob/50ca7e68034e179b2a8978ef62fd2a62464f7ceb/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L181): Define a constant instead of duplicating this literal "video" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
   private static final String PAUSE_CLASS_NAME = "pause-overlay";
   private static final String REPLAY_CLASS_NAME = "replay-overlay";
   private static final String CURRENT_TIME_CLASS_NAME = "current-time";
-  private static final String SPEAKER_CLASS_NAME = "speaker";
+  private static final String SPEAKER_CLASS_NAME = "volume-button,porvata-switchable-icon";
```
Comment (v3rth) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: I think it should looks like `volume-button.porvata-switchable-icon` buuutt... much cleaner will be change when you define SPEAKER_CLASS_NAME as `.volume-button.porvata-switchable-icon` with `.` at the beginng

Comment (bulaszenko) [reply src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: ups good catch itshould be "."

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L155](https://github.com/Wikia/selenium-tests/blob/ba433f96e42fb7310526211a5482adde7d390ab9/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L155): Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)
1. ![MINOR][MINOR] [AutoplayVuap.java#L181](https://github.com/Wikia/selenium-tests/blob/ba433f96e42fb7310526211a5482adde7d390ab9/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L181): Define a constant instead of duplicating this literal "video" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L155](https://github.com/Wikia/selenium-tests/blob/ecc9cc5acc2341a0d0a7526dc8bbf46de1493758/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L155): Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)
1. ![MINOR][MINOR] [AutoplayVuap.java#L181](https://github.com/Wikia/selenium-tests/blob/ecc9cc5acc2341a0d0a7526dc8bbf46de1493758/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L181): Define a constant instead of duplicating this literal "video" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L155](https://github.com/Wikia/selenium-tests/blob/d0f5d14904c9554b9dbb6718476c4a967403eae2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L155): Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)
1. ![MINOR][MINOR] [AutoplayVuap.java#L181](https://github.com/Wikia/selenium-tests/blob/d0f5d14904c9554b9dbb6718476c4a967403eae2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L181): Define a constant instead of duplicating this literal "video" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**